dask_deltatable/core.py

```diff
@@ -161,16 +163,20 @@ def read_delta_table(self, **kwargs) -> dd.core.DataFrame:
         pq_files = self.get_pq_files()
         if len(pq_files) == 0:
             raise RuntimeError("No Parquet files are available")
-        parts = [
-            delayed(
-                self.read_delta_dataset,
-                name="read-delta-table-" + tokenize(self.fs_token, f, **kwargs),
-            )(f, **kwargs)
-            for f in list(pq_files)
-        ]
-        meta = self._make_meta_from_schema()
-        verify_meta = kwargs.get("verify_meta", False)
-        return from_delayed(parts, meta=meta, verify_meta=verify_meta)
+
+        # Create Blockwise layer
+        label = "read-delta-table-"
+        output_name = f"{label}{tokenize(self.fs_token, **kwargs)}"
+        layer = DataFrameIOLayer(
```

Comment:
I *think* my general suggestion would be to use `from_map` instead of interacting directly with `DataFrameIOLayer` like this, but I may be missing something.

Reply:
Ref: https://blog.dask.org/2023/04/12/from-map

Reply:
Thanks @rjzamora. Looks like internally `from_map` is a wrapper for `DataFrameIOLayer`. The intent of this PR is to make sure we get nice blockwise optimization here. I've updated to use `from_map`. Let me know if that looks okay to you

Reply:
>from_map is a wrapper for DataFrameIOLayer

Yes exactly. All public IO functions (e.g. `read_parquet` and `read_csv`) use `from_map` for this purpose. `from_map` can be thought of as `DataFrameIOLayer`'s "public" API